pkg/restapi/oidc/operations.go

```diff
@@ -66,9 +77,17 @@ type Operation struct {
 
 // New returns a new Operation.
 func New(config *Config) (*Operation, error) {
+	// TODO make session cookies max age configurable: https://github.com/trustbloc/edge-agent/issues/388
+	cookieStore := sessions.NewCookieStore(config.Keys.Auth[:], config.Keys.Enc[:])
```

Comment:
case in point about preset byte array size in the struct's fields type, the sessions api takes []byte, not [32]byte as type. This is why you had to slice Auth and Enc. You don't need to use `[:]` if the size is not preset in the type.

Reply:
done